bindings/python/iota_sdk/types/token_scheme.py

```diff
@@ -28,6 +28,9 @@ class SimpleTokenScheme(TokenScheme):
     type: int = field(default_factory=lambda: 0, init=False)
 
     def as_dict(self):
+        """
+        Converts all integer values in the config to hexadecimal strings.
```

Comment:
These comments feel too specific, but I don't really care that much 🤷‍♀️

Reply:
I ended up not commenting it but I agree. I'd be fine with a generic comment for all as_dict methods tbh, these are way too specific and will break when/if we change fields.